src/main/cpp/t10_unique.cpp

```diff
@@ -25,5 +25,23 @@
 using namespace std;
 
 int t10_unique() {
-
+    int n;
+    bool f;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'f' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4493301)